src/components/GameFinder/GameFinder.css

```diff
@@ -0,0 +1,29 @@
+.GameFinder{
```

Comment:
`GameFinder`


Please, make the class names **cosistent**. You should use `lower case` for them


`.game-finder`